server/data-import/data.js

```diff
@@ -25,23 +22,36 @@ loadData()
 
 // note that exports are 'mongoexport --db hah-dev --collection pregnancycenters --jsonArray --out cessilye_nypc_geocoded.json'
 
-PregnancyCenterModel.collection.dropAsync()
-	.catch( (err) => log.error(err))
-	.then( () => {
-		fs.readFile('test/fixtures/cessilye_nypc.json', 'utf8', (err, data) => {
-			const docs = EJSON.parse(data)
-			log.info(typeof docs)
-			for (const doc of docs) {
+async function reimport() {
 
-				Joi.validateAsync(doc, pregnancyCenterSchemaJoi, {abortEarly: false})
-					.catch( (err) => log.error(err))
-					.then( (validatedData) => {
-						PregnancyCenterModel.createAsync(validatedData)
-							.then( (pc) => log.info(pc))
-					})
+	let docs
 
-			}
+	try {
+		await PregnancyCenterModel.collection.drop()
```

Comment:
That's a good point! I can take it out of the try/catch blocks or I can add an exit code within the catch. Do you know which is generally preferred?